opencog/atomspace/TruthValue.h

```diff
@@ -71,6 +71,17 @@ enum TruthValueType
     NUMBER_OF_TRUTH_VALUE_TYPES
 };
 
+//Flags for alternative ways of merging two different truth value types
+//as described in https://github.com/opencog/opencog/issues/1295
+enum TVMergeStyle
+{
+    USE_OLDER_TV_TYPE,
+    USE_NEWER_TV_TYPE,
```

Comment:
well, this is mis-named: merge does not change the TV type, it changes the TV.   The type of the TV will usually not change.

So really, it  should be this:
USE_OLDER_TV_TYPE  is KEEP_CURRENT_TV  // no-op, no change, do nothing return current TV
USE_NEWER_TV_TYPE is REPLACE_TV i // replace current TV with given TV  i.e. return given TV
USE_STRONGER_TV_TYPE, is STRONGER_TV // of the two, pick the stronger TV  which is what the current code does, and it is what Ben doesn't like

So none of these merge strategies do what PLN actually needs.